app/services/NpsService.scala

```diff
@@ -62,7 +62,7 @@ class NpsService @Inject()(repository: Repository, innerService: DirectNpsServic
 class DirectNpsService @Inject()(applicationConfig: ApplicationConfig, npsConnector: NpsConnector) {
   def getPayeATSData(nino: String, taxYear: Int)(
     implicit hc: HeaderCarrier): Future[Either[HttpResponse, PayeAtsMiddleTier]] =
-    npsConnector.connectToPayeTaxSummary(nino, taxYear, hc) map { response =>
+    npsConnector.connectToPayeTaxSummary(nino, taxYear - 1, hc) map { response =>
```

Comment:
This is a risky change, you are asking for taxyear-1 from nps but returning the calling code is expecting CY. Is this what's actually happening in QA/Prod?

Reply:
It would not cause any issues. Since now we are using CY as 2020 instead of 2019 and we need to do CY-1 to get data on QA/Prod.